apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1732,7 +1732,7 @@ export const local_development: NavMenuConstant = {
       name: 'Local development',
       url: undefined,
       items: [
-        { name: 'Local development', url: '/guides/local-development/overview' },
+        { name: 'Schema migrations', url: '/guides/local-development/schema-migrations' },
```

Comment:
```suggestion
        { name: 'Local development', url: '/guides/local-development/overview' },
```

I propose reverting this change, reasoning: we have a good chunk of users who are not familiar with a schema migration workflow, and naming this "schema migrations" pretty much guarantees they won't read it. I think if we call the document "overview", but in the actual document, stress that "schema migrations _are_ the way to do local development", we'll be more likely to educate users.

Reply:
Actually the `Local development` title is repeated twice in the nav bar.

![image](https://github.com/user-attachments/assets/7b58ce39-9f72-4311-888a-029497242a58)

How about naming it `Getting started` to be consistent with CLI > Getting started or simply `Overview`?

Reply:
Ooops completely forgot that one existed. I like Getting Started!